ra2ce/analysis/direct/effectiveness_measures.py

```diff
@@ -28,32 +7,45 @@
 import pandas as pd
 
 from ra2ce.analysis.analysis_config_data.analysis_config_data import (
-    AnalysisConfigData,
     AnalysisSectionDirect,
 )
-
-
-class EffectivenessMeasures:
-    """This is a namespace for methods to calculate effectiveness of measures"""
-
-    def __init__(self, config: AnalysisConfigData, analysis: AnalysisSectionDirect):
+from ra2ce.analysis.direct.analysis_direct_protocol import (
+    AnalysisDirectProtocol,
+)
+from ra2ce.network.graph_files.network_file import NetworkFile
+
+
+class EffectivenessMeasures(AnalysisDirectProtocol):
+    graph_file: NetworkFile
+    analysis: AnalysisSectionDirect
+    input_path: Path = None
+    output_path: Path = None
+    result: gpd.GeoDataFrame = None
```

Comment:
Same as in previous comment. When having an `__init__` this might not be the best place to define default values.